### PR TITLE
Create desktop version of the proto #2

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Turning the phone version into a desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,94 +61,102 @@ <h2 class="LETS">LET’S CONNECT</h2>
     <section class="works">
```
Comment (D4nielJ) [inline index.html:61]: - [ ] Let's fix the layout in the works section, if you notice in the Figma template each one of the cards has the image on different sides of the card. Let's recreate this effect. To achieve this effect, I recommend you add a modifier class to the pair cards.

image

While we're at it, would you mind personalizing the info and the images of each one of the cards so we get a better idea of how they behave with larger titles, descriptions, different techs, etc.

File: index.html [hunk 2]
```diff
@@ -203,51 +215,49 @@ <h3>Skills</h3>
     </div>
     </section>
 
-    <section class="form-wrapper">
-      <div class="form-title">
+    <footer>
```
Comment (D4nielJ) [inline index.html:218]: - [ ] Let's add the bg from the contact-me section as well. Is currently missing.

image

File: index.html [hunk 3]
```diff
@@ -57,94 +61,102 @@ <h2 class="LETS">LET’S CONNECT</h2>
...
+              <li class="ppp">html</li>
+              <li class="ppp">css</li>
+              <li class="ppp">javaScript</li>
+            </ul>
+            <button type="button">See Project</button>
+          </div>
         </div>
       </div>
     </section>
     <section class="about-me">
```
Comment (D4nielJ) [inline index.html:157]: - [ ] Let's revisit the mobile version of the `about-me` section. Translating to a desktop version should have the minimum, ideally, zero effects in our mobile version. I think some margins are missing from this section now.

image

- [ ] Let's also fix the Medium icon from this section, currently, is rendering smaller than the other icons.

image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @eng-mohamed-eldeeb, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥 🔥 🔥 

- Overall, you've made a magnificent work with your UI 👏 
- You followed GitHub flow correctly and your linters checked.

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eng-mohamed-eldeeb** [comment]
I have done all reviewer required changes.

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @eng-mohamed-eldeeb 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**